src/js/index.js

```diff
@@ -85,7 +94,7 @@ class RegularTableElement extends RegularViewEventModel {
      * on the next draw() call.
      *
      * @internal
-     * @private
+     * @protected
```

Comment:
I marked this as `@protected` to help out with some experimental use in `tree-finder`. This should maybe actually be `@public`, since `_resetAutoSize` needs to be called manually in at least [some cases](https://github.com/jpmorganchase/regular-table/blob/master/examples/file_browser.md#ui). Would that change make sense to you?